torch/nested/_internal/nested_tensor.py

```diff
@@ -422,5 +360,22 @@ def jagged_from_tensor_and_lengths(
     return (ret_nt, offsets, None if is_contiguous else length_list)
 
 
-def buffer_from_jagged(jagged):
-    return ViewBufferFromNested.apply(jagged)
+# NB: A dummy arg is required so that NestedTensor.__torch_dispatch__() is invoked
+# for _nested_view_from_values_offsets(). Sizes don't matter much, but they shouldn't be
+# 0/1 because the dummy can be fake-ified and we want to avoid specializing.
+# This arg is otherwise unused.
```

Comment:
Didn't we come up with some BackendSelect like strategy for avoiding the dummy arg?

Reply:
Not that I remember, but that would be awesome if we could avoid it